### DISCUSSION TITLE
Save file encrypted?

### DISCUSSION THREAD

**Discussion by Finnareal** [dialogic-godot/dialogic#2196] [Q&A] [answered]
Hello!
I have empty Encryption Password field in settings, but my save file still looks like it is encrypted. And my info file looks the same way. So I can't check if my game info saved correct. Should I change the encoding somehow or is the file actually encrypted?![image_2024-04-20_193727462](https://github.com/dialogic-godot/dialogic/assets/32553595/3eff500b-f525-4f96-b714-60f31ebda642)?

**Comment by CakeVR** [comment, marked as answer]
Hello, that file is a binary file and not encrypted.
